styles.css

```diff
@@ -225,21 +225,23 @@ input {
 }
 
 #actual-timer-container {
-  align-items: center;
   display: flex;
   flex-direction: column;
   background-color: #46424D;
-  height: 68%;
```

Comment:
This is a big change - I hope it resolved the issues we were having with the bar at the bottom of the section.